src/pages/store/collection/[uuid].tsx

```diff
@@ -11,35 +11,53 @@ import styles from '@/styles/pages/StoreCollectionPage.module.scss';
 import { GetServerSideProps } from 'next';
 
 interface CollectionProps {
+  uuid: string;
   user: PrivateProfile;
   collection: PublicMerchCollection;
+  previewPublic: boolean;
 }
 
 const CollectionsPage = ({
-  user: { credits },
-  collection: { title, description, items = [] },
+  uuid,
+  user: { credits, accessType },
+  collection: { title, description, items = [], archived },
+  previewPublic,
 }: CollectionProps) => {
+  const canManageStore = PermissionService.canEditMerchItems.includes(accessType) && !previewPublic;
+
   return (
     <div className={styles.container}>
       <Navbar balance={credits} showBack />
       <div className={styles.header}>
         <Typography variant="h1/bold" component="h1">
           {title}
+          {canManageStore && <EditButton type="collection" uuid={uuid} />}
```

Comment:
i might have missed this syntax existing in a few previous PRs but we actually have a standard on this for conditional rendering where we should always write it as 
 ```suggestion
          {canManageStore ? <EditButton type="collection" uuid={uuid} /> : null}
```

the bug this is avoiding is where you have a check such as 
```
const quantity = 0;
{quantity && <MyComponent />}
```
you would expect this not to render because the conditional is falsy however javascript shortcircuiting will still return the first value in this case so react will render a plaintext 0 on the screen. by writing it with the suggested syntax, this bug is not possible since the conditional is never the returned value